devel/ansible/roles/dev/tasks/rabbitmq.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Install RabbitMQ packages
+  package:
+      name: "{{ item }}"
+      state: present
+  with_items:
+      - rabbitmq-server
+
+- name: Create RabbitMQ systemd override directory
+  file:
+    path: /etc/systemd/system/rabbitmq-server.service.d/
+    state: directory
+
+- name: Override rabbitmq hostname
```

Comment:
It worked without this step for me. Can you try again and if it fails again maybe we can debug it together?